dist/source/repo-creator/choose-name.css

```diff
@@ -55,3 +55,24 @@
 	border-radius:6px;
 	float:right;
 }
+
+.choose-name .has-warning .repo-input{
+	border-color: #8a6d3b;
+    -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, .075);
+    box-shadow: inset 0 1px 1px rgba(0, 0, 0, .075);
+}
+
+/*** validation ****/
+p.aurelia-validation-message {
+  display: inline;
+  margin-left: 5px;
+}
+p.aurelia-validation-message:empty {
+  display: none;
+}
+.has-success label {
+  color: #3c763d;
+}
+.has-warning label {
+  color: #8a6d3b;
```

Comment:
Looks like the first part of this chunk was fixed (tabs instead of spaces) but the rest of it still has spaces.  You can tell which by the width in the Pull Request.